conda/recipes/ptxcompiler/meta.yaml

```diff
@@ -13,16 +14,23 @@ source:
 
 build:
   number: {{ GIT_DESCRIBE_NUMBER }}
-  string: cuda_{{ cuda_version }}_py{{ py_version }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+  string: cuda_{{ cuda_major }}_py{{ py_version }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+  host:
     - python
     - pip
+    - cudatoolkit {{ cuda_version }}.*
   run:
     - python
     - numba >=0.54
+    - cudatoolkit >={{ cuda_major ~ ".0" }},<={{ cuda_version }}
```

Comment:
Just a note based on a conversation Graham and I had offline, this only constrains to the minor version used (not patch), but that is ok as patch version should not impact `ptxcompiler` per [this doc]( https://docs.nvidia.com/cuda/parallel-thread-execution/index.html#release-notes__ptx-release-history ). IOW nothing to do here just recording for posterity 🙂